Describe potenatial problems in `topic_pattern` setting (#122)
logstash-plugins/logstash-integration-kafka: docs/input-kafka.asciidoc

  * Value type is <<string,string>>
  * There is no default value for this setting.

A topic regular expression pattern to subscribe to.

Filtering by a regular expression is done by retrieving the full list of topic names from the broker and applying the pattern locally. When used with brokers with a lot of topics this operation could be very slow, especially if there are a lot of consumers.

NOTE:  When the broker has some topics configured with ACL rules and they miss the DESCRIBE permission, then the subscription

> happens as expected but on broker side is logged the fact that it denied to describe some topics from the Logstash's user.

Reviewer: ```suggestion
happens but on the broker side it is logged that the subscription of some topics was denied to the configured user.
```

  * Value type is <<string,string>>
  * There is no default value for this setting.

> A topic regex pattern to subscribe to; retrieve the full list from the broker and filters applying the pattern locally.
> The <<plugins-{type}s-{plugin}-topics>> configuration will be ignored when using this configuration.
> 
> NOTE: When used with brokers with a lot of topics, this operation, could be very slow especially if there are a lof of consumers.

Reviewer: ```suggestion
A topic regular expression pattern to subscribe to.

Filtering by a regular expression is done by retrieving the full list of topic names from the broker and applying the pattern locally. When used with brokers with a lot of topics this operation could be very slow, especially if there are a lot of consumers.
```

NOTE: When the broker has some ACL rules, configured with WRITE or DESCRIBE permissions, there are some unexpected access permissions

> on topics that doesn't match the regex pattern. This is due to the fact that the full list of topics has to be retrieved.

Reviewer: Can we make it clearer to the user what happens in this case? Logstash will retrieve the full list of topics, and some will have ACLs. Does it mean that topics Logstash can't access are skipped? is there an error shown? does the pipeline crash?

Author: :+1:  the subscription happens has expected but the broker logs line as 
```
Principal = User:<Logstash username> is Denied Operation = Describe from host ...
```
